### DISCUSSION TITLE
Maybe add deprecation of Matomo 3.x to the README

### DISCUSSION THREAD

**Discussion by EPNW-Eric** [Floating-Dartists/matomo-tracker#84] [Ideas]
As mentioned in #75 by @TesteurManiak there are still some url parameters that were removed in Matomo 4. While I think it's ok to have them (and thus Matomo 3.x support) for the next major release (which should happen in a bit?), the next major release after that should/could remove the parameters. If that's the plan we should in preparation therefore add a note to the README that Matomo 3.x support will be dropped in the future.

Edit: I did a quick internets search but was not able to find information on the version distribution, so I asked on the [Matomo forum](https://forum.matomo.org/t/matomo-version-distribution/51248).

Edit 2: Oh and I forgot to mention that I personally would be fine with already dropping 3.x support in the next major release, but that's just because we're only using Matomo 4+ internally.

**Comment by TesteurManiak** [comment]
I agree we should add a notice deprecation in the README and prepare to drop support for 3.x.x. I don't think we'll drop it in v4 but we can plan to remove it in v5 of the package.
